src/signals/incident/components/form/MapSelectors/Asset/Selector/WfsLayer/AssetLayer/AssetLayer.tsx

```diff
@@ -179,12 +185,19 @@ export const AssetLayer: FunctionComponent<DataLayerProps> = ({
           'click',
           /* istanbul ignore next */ () => {
             const { description, typeValue, idField } = featureType
+
+            if (typeValue === 'reported') {
+              return
+            }
+
             const coordinates = featureTolocation(feature.geometry as Geometrie)
+
             const item: Item = {
               // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
               id: feature.properties[idField]!,
               type: typeValue,
               description,
+              isReported: Boolean(feature.properties.meldingstatus === 1),
```

Comment:
No need to wrap it in a `Boolean` expression since the strict comparison already is a boolean.

Reply:
✅